examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImWriteCommand.java

```diff
@@ -7,12 +7,37 @@
 import chip.devicecontroller.model.ChipAttributePath;
 import com.matter.controller.commands.common.CredentialsIssuer;
 import java.util.ArrayList;
+import java.util.logging.Logger;
 import java.util.logging.Level;
+import javax.annotation.Nullable;
 
-public final class PairOnNetworkLongImWriteCommand extends PairingCommand
-    implements WriteAttributesCallback, GetConnectedDeviceCallback {
+public final class PairOnNetworkLongImWriteCommand extends PairingCommand implements GetConnectedDeviceCallback{
   private static final int MATTER_PORT = 5540;
   private long devicePointer;
+  private static final int CLUSTER_ID_BASIC = 0x0028;
+  private static final int ATTR_ID_LOCAL_CONFIG_DISABLED = 16;
+  private static Logger logger = Logger.getLogger(PairOnNetworkLongImWriteCommand.class.getName());
+
+  private class InternalWriteAttributesCallback implements WriteAttributesCallback {
```

Comment:
The same could be done for `GetConnectedDeviceCallback` btw

Reply:
done, thanks